omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PersonAttributeTypeController1_8Test.java

```diff
@@ -153,6 +154,25 @@ public void retirePersonAttributeType_shouldRetireAPersonAttributeType() throws
 		Assert.assertTrue("unit test".equals(obj.getRetireReason()));
 	}
 	
+	@Test
+	public void shouldUnRetireAPersonAttributeType() throws Exception {
+		
+		final String nonRetiredAttribute = "a0f5521c-dbbd-4c10-81b2-1b7ab18330df";
+		PersonAttributeType obj = service.getPersonAttributeTypeByUuid(nonRetiredAttribute);
```

Comment:
What does obj mean?

Reply:
Just a variable name. Meaning comes from its type definition → `PersonAttributeType` .

Reply:
Can you do some googling about good variable names?

Reply:
Thank you @dkayiwa , this helped as it usually does. I going to rename this for better meaning.